pkg/endpoint/endpoint.go

```diff
@@ -2093,14 +2094,18 @@ func (e *Endpoint) UpdateProxyStatistics(l7Protocol string, port uint16, ingress
 	}
 
 	stats.Received++
+	metrics.ProxyReceived.Inc()
 
 	switch verdict {
 	case accesslog.VerdictForwarded:
 		stats.Forwarded++
+		metrics.ProxyForwarded.Inc()
 	case accesslog.VerdictDenied:
 		stats.Denied++
+		metrics.ProxyDenied.Inc()
 	case accesslog.VerdictError:
```

Comment:
Note that this code branch is dead. Kafka is never emitting log messages with this verdict. That needs to be fixed in the Kafka proxy.

Reply:
Please add unit tests for all types of verdicts to check that the metrics get incremented as expected.

Reply:
@rlenglet Raised #4041 to fix the Kafka parser. I will do that as a separate PR. Thanks! Will add tests in this PR.

Reply:
@rlenglet raised #4043 for adding the CI tests as there is not a non-generic way of unit testing this. Thanks.